process/transaction/export_test.go

```diff
@@ -102,7 +102,7 @@ func (inTx *InterceptedTransaction) CheckMaxGasPrice() error {
 
 // VerifyGuardian calls the un-exported method verifyGuardian
 func (txProc *txProcessor) VerifyGuardian(tx *transaction.Transaction, account state.UserAccountHandler) error {
```

Comment:
there is no need for this export now

Reply:
Indeed, removed (actually, by mistake, I did a bad recursive call).